packages/fluentui/react-northstar/src/components/Button/Button.tsx

```diff
@@ -346,6 +353,7 @@ Button.propTypes = {
   loading: PropTypes.bool,
   onClick: PropTypes.func,
   onFocus: PropTypes.func,
+  tinted: customPropTypes.every([customPropTypes.disallow(['secondary']), PropTypes.bool]),
```

Comment:
I think this means that `tinted` cannot be used with the `secondary` button is that a correct udnerstanding ?

Reply:
Correct